locale/en/corteza-webapp-admin/system.queues.yaml

```diff
@@ -43,4 +43,4 @@ list:
       label: Including
   loading: Loading messaging queues
   new: New
-  title: Messaging queues
```

Comment:
Same here, revert please. (You removed the last line of the file, and files need empty lines at the end)